src/libraries/System.Text.Json/ref/System.Text.Json.cs

```diff
@@ -142,8 +142,8 @@ public partial class JsonException : System.Exception
         protected JsonException(System.Runtime.Serialization.SerializationInfo info, System.Runtime.Serialization.StreamingContext context) { }
         public JsonException(string? message) { }
         public JsonException(string? message, System.Exception? innerException) { }
-        public JsonException(string? message, string path, long? lineNumber, long? bytePositionInLine) { }
-        public JsonException(string? message, string path, long? lineNumber, long? bytePositionInLine, System.Exception? innerException) { }
+        public JsonException(string? message, string? path, long? lineNumber, long? bytePositionInLine) { }
+        public JsonException(string? message, string? path, long? lineNumber, long? bytePositionInLine, System.Exception? innerException) { }
```

Comment:
We had lots of conversation about Path nullability then ended up with making nullable https://github.com/dotnet/runtime/pull/528#discussion_r361762878